lock.go

```diff
@@ -53,7 +53,7 @@ func readLock(r io.Reader) (*lock, error) {
 	for i, ld := range rl.P {
 		r := gps.Revision(ld.Revision)
 
-		var v gps.Version
+		var v gps.Version = r
```

Comment:
@mvdan, this change created a new golint warning:

```
lock.go:56: should omit type gps.Version from declaration of var v; it will be inferred from the right-hand side
```

There's one more on status.go:97 FWIW. See https://go-lint.appspot.com/github.com/golang/dep.